src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionCategoryService.java

```diff
@@ -1,18 +1,23 @@
 package pl.byczazagroda.trackexpensesappbackend.service;
 
-
 import java.util.List;
 
 import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
 
 import org.springframework.web.bind.annotation.RequestBody;
 
 import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryCreateDTO;
 import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryUpdateDTO;
 
 
 public interface FinancialTransactionCategoryService {
```

Comment:
Please find out if the "@Validated" annotation should also be added to the service interface.